infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,88 @@
 
 ## Why do we need it?
-You should use the Registry if you want to:
-- tightly control where your images are being stored
-- fully own your images distribution pipeline
-- integrate image storage and distribution tightly into your in-house development workflow
+You should use the Harbor Registry if you want to:
+- tightly control where your images are being stored.
+- fully own your images distribution pipeline.
+- integrate image storage and distribution tightly into your in-house development workflow.
 - leverage the high-speed network that connects your servers, avoiding to consume precious Internet bandwidth to transfer images stored in the Docker Hub public service.
+- leverage [Proxy Cache](https://goharbor.io/docs/2.4.0/administration/configure-proxy-cache/) functionalities, to not exceed Docker Hub’s rate limiting policy.
+- have a [vulnerability scanner](https://goharbor.io/docs/2.4.0/administration/vulnerability-scanning/) to detect possible image vulnerabilities.
+- manage your [Helm Charts](https://goharbor.io/docs/edge/working-with-projects/working-with-images/managing-helm-charts/).
 
 Finally, consider that, in this Kubernetes setup, users instantiate mainly VMs, whose image may be rather large. Allowing users to download the VM image locally, instead of from a remote server, would greatly impact on their quality of experience in term of time required to start their service.
 
-## Docker Registry Helm Chart
-We used the [Docker Registry Helm Chart](https://github.com/helm/charts/tree/master/stable/docker-registry), that is a Kubernetes chart to deploy a private Docker Registry where we have appropriately modified the values in [values.yaml](https://github.com/helm/charts/blob/master/stable/docker-registry/values.yaml) file.
+## Harbor Registry Helm Chart
+To install Harbor, it is possible to leverage the [official Helm Chart](https://github.com/helm/charts/tree/master/stable/docker-registry), appropriately configuring the `values.yaml` file (additional details follow in the next sections).
 
-### Configuration
-These are our modification of the various file.
+### Pre-requisites
 
-  1. Annotations concerning the authentication in the Ingress controller.
-```yaml
-annotations:
-   nginx.ingress.kubernetes.io/auth-realm: Authentication Required - ok
-   nginx.ingress.kubernetes.io/auth-secret: basic-auth
-   nginx.ingress.kubernetes.io/auth-type: basic
-```
+  1. [Kubernetes cluster 1.10+](https://kubernetes.io/releases/download/)
+  2. [Helm 3](https://helm.sh/docs/intro/install/)
+  3. [High available ingress controller](https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/ingress-controller) (Harbor does not manage the external endpoint)
+  4. [High available PostgreSQL 9.6+](https://github.com/netgroup-polito/CrownLabs/tree/master/infrastructure/identity-provider#postgresql-operator) (Harbor does not handle the deployment of HA of database)
```

Comment:
Would remove all those links (but the one for Helm), and move the postgres and redis one directly to the appropriate section.